Mathlib/RingTheory/Localization/Away/Basic.lean

```diff
@@ -61,6 +61,18 @@ theorem mul_invSelf : algebraMap R S x * invSelf x = 1 := by
   apply IsLocalization.mk'_one
 #align is_localization.away.mul_inv_self IsLocalization.Away.mul_invSelf
 
+/-- For `s : S`, this is a choice of `(x, n) : R × ℕ` such that
+`s * algebraMap R S (s ^ n) = algebraMap R S r`. -/
```

Comment:
Where is `x` in the statement? What is `r`?
Also, `x` is a `variable` here, so the name `x` should not be used to describe something else.
Did you mean to say
"For `s : S`, this is a choice of `(r, n) : R × ℕ` such that `s * algebraMap R S (x ^ n) = algebraMap R S r`?
(I.e., expressing `s` as `r/x^n` informally.)